### PR TITLE
Error handling single ticket

### PR DESCRIPTION
**Pull request by ColinStClaire**
ready to merge into http-client!

### CODE REVIEW

File: TicketDesk/ngWebClient-StandAlone/src/app/services/single-ticket.service.ts [hunk 1]
```diff
@@ -20,10 +20,7 @@ interface EventList {
 @Injectable()
 export class SingleTicketService {
 
-  constructor(
-    private http: HttpClient
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/services/single-ticket.service.ts:24]: I'll bet you anything this spacing was me.... YIKES

File: TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts [hunk 2]
```diff
@@ -1,18 +1,23 @@
 import { AdContactService } from './../services/ad-contact.service';
 import { UserDetails } from 'app/models/user-details';
 import { Component, OnInit, Input } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { NUMBER_TYPE } from '@angular/compiler/src/output/output_ast';
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts]: Not sure what this is being used for and it's making me kind of nervous

File: TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts [hunk 3]
```diff
@@ -1,18 +1,23 @@
 import { AdContactService } from './../services/ad-contact.service';
 import { UserDetails } from 'app/models/user-details';
 import { Component, OnInit, Input } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts]: This should be unnecessary

File: TicketDesk/ngWebClient-StandAlone/src/app/services/ad-contact.service.ts [hunk 4]
```diff
@@ -11,9 +11,25 @@ import * as settings from '../app-settings';
...
+      firstName: '<First Name>',
+      lastName: '<Last Name>',
+      phoneNumber: '<Phone Number>',
+      email: 'stillneed@activedirectory.com',
+      userId: userID});
+  }
+
+  private handleError(error: HttpErrorResponse): ErrorObservable {
+    if (error.error instanceof ErrorEvent) {
+      // ... this is a client side error, handle it!
```
Comment (kixiQu) [inline TicketDesk/ngWebClient-StandAlone/src/app/services/ad-contact.service.ts:27]: I don't get how general this is supposed to be. Is it going to handle all different kinds of errors? Actually, looking at it again, I guess it's pretty much the sample code. No changes needed, but probably worth talking eventually about where different responses for different problems will be differentiated.

**Review by kixiQu — CHANGES_REQUESTED** [review]
Remove unnecessary imports if unnecessary, explain that NUMBER_TYPE thing to me if we're keeping it, and otherwise we should be good

File: TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts (hunk 3 above)
Comment (ColinStClaire) [reply TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts]: removed

File: TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts (hunk 2 above)
Comment (ColinStClaire) [reply TicketDesk/ngWebClient-StandAlone/src/app/contact-info/contact-info.component.ts]: removed

File: TicketDesk/ngWebClient-StandAlone/src/app/services/ad-contact.service.ts (hunk 4 above)
Comment (ColinStClaire) [reply TicketDesk/ngWebClient-StandAlone/src/app/services/ad-contact.service.ts:27]: yeah it needs to be continued in a new branch for an error handler class that can handle all of our services

**Review by JoschaO — APPROVED** [review]
Okay it looks ready

**Pull request merged**